include/robot_interfaces/robot_backend.hpp

```diff
@@ -177,11 +201,29 @@ class RobotBackend
     {
         real_time_tools::set_cpu_dma_latency(0);
```

Comment:
side note: i think we could get rid of this, as by default this is done when starting a realtime thread, right?

Reply:
I just checked, yes this is already done in `create_realtime_thread`.  I'll remove it everywhere but will do so in a separate PR.